automatic/sysinternals/tools/helpers.ps1

```diff
@@ -1,5 +1,6 @@
 function Accept-Eula() {
 
+$ShellRunas = 'ShellRunas - Sysinternals: www.sysinternals.com'
```

Comment:
What is the reason you have this as a separate variable instead of inlining it like the other ones are?

Reply:
It was to ease the formatting of the array. I couldn't quite get satisfied trying to keep the array columns in sync and I wanted to try to make the diff easier to read.

Reply:
Fair enough, maybe someone else should take a look at the formatting.

Reply:
Gotcha. I don't mind in-lining it in the array and doing the spacing work if that would be more ideal. We could also add an `if` block within the `foreach` but I thought that would be too much extra code.